### ISSUE TITLE
shiny 1.3 breaks HTML UI (which works in shiny 1.2)

### ISSUE THREAD

**Issue by pvermees** [rstudio/shiny#2380]
I have an html-only UI that worked fine with shiny 1.2.0 but breaks in 1.3.0:

Installation:

```
library(devtools)
install_github('pvermees/IsoplotR')
install_github('pvermees/IsoplotRgui')
```

Running the app:

```
library(IsoplotRgui)
IsoplotR()
```

When I run this with shiny 1.3.0, the browser shows the following error message:

**No UI defined
Shiny couldn't find any UI for this application. We looked in:
www/index.html
ui.R
app.R**

**Comment by schloerke** [comment]
@pvermees I can repro the issue.  

Quick work around while a fix is implemented: https://github.com/pvermees/IsoplotRgui/pull/36

**Comment by wch** [comment]
For future reference: the workaround is to replace the server.R file with app.R, and do this:

```R
server <- function(input, output, session) {
   # Existing server code here
}

shinyApp(ui = htmlTemplate("www/index.html"), server)
```

**Comment by pvermees** [comment]
Thanks for the workaround, which I confirm fixes the issue. But out of interest: will the old version work again with future versions of shiny?

**Comment by schloerke** [comment]
@pvictor The old behavior should be fixed in #2380

**Comment by wch** [comment]
The fix is currently available on the `v1.3-patch` branch.

```R
devtools::install_github('rstudio/shiny@v1.3-patch')
```
